web/pdf_attachment_viewer.js

```diff
@@ -116,13 +116,12 @@ class PDFAttachmentViewer {
   /**
    * @param {PDFAttachmentViewerRenderParameters} params
    */
-  render(params = {}) {
-    var attachments = params.attachments || null;
-    var attachmentsCount = 0;
+  render({ attachments, keepRenderedCapability = false, }) {
+    attachments = attachments || null;
```

Comment:
I think we can remove this line and just set `attachments = null` in the destructured object above.